pkg/ssh/connection.go

```diff
@@ -273,31 +271,39 @@ func (c *connection) Close() error {
 	return c.sshclient.Close()
 }
 
-func (c *connection) Stream(cmd string, stdout io.Writer, stderr io.Writer) (int, error) {
+func (c *connection) POpen(cmd string, stdin io.Reader, stdout io.Writer, stderr io.Writer) (int, error) {
```

Comment:
The function name might not be the clearest. What does `P` refer to? Does the name refer to the `popen` syscall? You should add a comment describing this.